User Guide refactor (#1795)
zowe/docs-site: docs/user-guide/install-overview.md

# Overview

> The installation of Zowe&trade; consists of two independent processes: installing the Zowe server components either entirely on z/OS or a combination of z/OS and Docker, and then installing Zowe CLI on a desktop computer.

Reviewer: ```suggestion
The installation of Zowe&trade; consists of the following processes:
- installation of the Zowe server components.

   You can install the components either on z/OS only or you can install some components on z/OS and on Docker.
   
- installation of Zowe CLI on a desktop computer.
```

Reviewer 2: ```suggestion
The installation of Zowe&trade; consists of the following processes:
- installation of the Zowe server components either on z/OS only or on z/OS and Docker.
- installation of Zowe CLI on a desktop computer.
```

> The Zowe server components provide a web desktop that runs in a web browser providing a number of applications for z/OS users, together with an API mediation layer provides single-sign on (SSO), organization of the multiple zowe servers under a single website, and other capabilities useful for z/OS developers. 

Reviewer: ```suggestion
The Zowe server components provide a web desktop that runs a number of applications such as API Mediation Layer that includes the Single Sign-on (SSO) capability, organization of the multiple Zowe servers under a single website, and other useful features for z/OS developers. 
```

Reviewer 2: ```suggestion
The Zowe server components provide:
- a browser-based desktop that runs a number of applications.
- the API Mediation Layer which includes the Single Sign-on (SSO) capability.
- organization of the multiple Zowe servers under a single website.
- other useful features for z/OS developers. 
```

> Because Zowe is a set of components, before installing Zowe, first determine which components you want to install and where you want to install them. This guide provides documentation for all of the components and it is split into sections so you can install as much as you need.

Reviewer: ```suggestion
Because Zowe is a set of components, before installing Zowe, use this guide to determine which components you want to install and where you want to install them.
```

> Here are some scenarios to consider:

Reviewer: ```suggestion
You might consider the following scenarios:
```

Reviewer 2: Just say, "Consider the following scenarios:"?

Author: I changed it to "Consider the following scenarios:" for more simplicity. :)

> - If you will only be using the Zowe CLI on PCs, depending on the plugins being used, you may not need to install the Zowe server components. If you are just using core Zowe CLI groups from your PC, the z/OS you connect to does not required any components of Zowe to be installed on z/OS, unless you want to take advantage of advanced authentication methods such as single sign-on or multi-factor authentication.

Reviewer: ```suggestion
- If you plan to use Zowe CLI on PC only, you may not need to install the Zowe server components.
**Note**: Some CLI plug-ins require the installation of components on z/OS.
If you plan to use core Zowe CLI groups from your PC, the z/OS you connect to does not require any components of Zowe to be installed on z/OS, unless you want to take advantage of advanced authentication methods such as single sign-on or multi-factor authentication.
```

> - If you are using the Docker technical preview to run the linux parts of Zowe in a container, then you only need to configure Zowe's z/OS component to start the ZSS server.

Reviewer: ```suggestion
- If you use the Docker technical preview to run the Linux parts of Zowe in a container, you only need to configure the Zowe z/OS component to start the ZSS server.
```
